vertx-web-graphql/src/main/java/examples/GraphQLExamples.java

```diff
@@ -80,11 +82,37 @@ public void handlerSetupBatching(GraphQL graphQL) {
     GraphQLHandler handler = GraphQLHandler.create(graphQL, options);
   }
 
+  public void setupGraphQLHandlerMultipart() {
+    GraphQLHandler graphQLHandler = GraphQLHandler.create(
+      setupGraphQLJava(),
+      new GraphQLHandlerOptions().setRequestMultipartEnabled(true)
+    );
+
+    Router router = Router.router(Vertx.vertx());
+
+    router.route().handler(BodyHandler.create());
+    router.route("/graphql").order(100).handler(graphQLHandler);
+  }
+
+  public void setGraphqlMultipart() {
```

Comment:
I believe this example can be removed. It's already shown in `setupGraphQLHandlerMultipart`. So removing this and rewording the doc should be enough.